CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 ## :telephone_receiver: Open Calls
 
-:clock10: **When?** Every Saturday at 19:00 UTC (noon PT / 3pm ET / [your timezone][]) \
+:clock10: **When?** Every Saturday at 20:00 GMT (noon PT / 3pm ET / [your timezone][]) \
```

Comment:
```suggestion
:clock10: **When?** Every Saturday at 20:00 GMT (noon PT / 3pm ET / 4am Taiwan / [your timezone][]) \
```

Reply:
Might as well make the timezones a bit more inclusive in spirit, if not in temporality rn :)

Reply:
```suggestion
:clock10: **When?** Every Saturday at 20:00 UTC (noon PT / 3pm ET / [your timezone][]) \
```